packages/table/README.md

```diff
@@ -141,8 +138,7 @@ When `selects="multiple"`, the `<sp-table>` will manage a selection in the array
 
 ```html
```

Comment:
I don't know why it's not letting me leave a comment on line 137, but anyways... while you're working on this PR, could you fix the first sentence in that line for me? 😅 

> When `selects="multiple"`, the `<sp-table>` manages selections via a presence toggle and adds them to the `selected` array.